dev-vcs/git-filter-repo/git-filter-repo-2.34.0.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=8
 
 PYTHON_COMPAT=( python3_{9..10} )
```

Comment:
Given this affects installed files, I'd revbump the older versions (just git mv if you want), or just only bother touching 2.38.0.

Reply:
Not 100% sure what you're saying, but I squashed in a rename of these to `2.3{3,4}.0-r1`.  IIUC that'll trigger a rebuild for users.